src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -0,0 +1,415 @@
+    pal_record_a['fipsCode'] = 20063955100
+    pal_record_b['fipsCode'] = 20063955200
+
+    # Set the PAL frequency.
+    pal_low_frequency = 3570000000
+    pal_high_frequency = 3580000000
+
+    # Load device info.
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Set the same user ID for all devices
+    device_b['userId'] = device_a['userId']
+
+    # Device_a is Category A.
+    self.assertEqual(device_a['cbsdCategory'], 'A')
+
+    # Device_b is Category B with conditionals pre-loaded.
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+
+    # Set the values of fipsCode in pal_records_a and b to make them adjacent.
+    # 20063955100 and 20063955200 respectively
+    pal_records = makePalRecordsConsistent([pal_record_a, pal_record_b],
+                                           pal_low_frequency, pal_high_frequency,
+                                           device_a['userId'])
+
+    # Set the locations of devices to reside with in service area
+    device_a['installationParam']['latitude'], device_a['installationParam'][
+        'longitude'] = 39.0373, -100.4184
+    device_b['installationParam']['latitude'], device_b['installationParam'][
+        'longitude'] = 39.0378, -100.4785
+
+    # Set the AntennaGain and EIRP capability
+    device_a['installationParam']['eirpCapability'] = 30
+    device_b['installationParam']['eirpCapability'] = 47
+    device_a['installationParam']['antennaGain'] = 16
+    device_b['installationParam']['antennaGain'] = 16
+
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam'],
+        'measCapability': device_b['measCapability']
+    }
+    conditionals = [conditionals_b]
+    del device_b['installationParam']
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+    del device_b['measCapability']
+
+    # Create the actual config.
+    devices = [device_a, device_b]
+    config = {
+        'registrationRequests': devices,
+        'conditionalRegistrationData': conditionals,
+        'palRecords': pal_records,
+        'sasTestHarnessCert': os.path.join('certs', 'sas.cert'),
+        'sasTestHarnessKey': os.path.join('certs', 'sas.key')
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_PCR_1_default_config)
+  def test_WINNF_FT_S_PCR_1(self, config_filename):
+    """ Successful Maximum PPA Creation.
+
+    Checks PPA generated by SAS UUT shall be fully contained within the service area.
+    """
+
+    # Load the Config file
+    config = loadConfig(config_filename)
+
+    # light checking of itu,terrain and landcover data path exists.
+
```

Comment:
Do these tests belong here?
These tests are basically that your test harness environment is well setup.

Shouldnt they be instead in the sas_testcase parent class, or somewhere similar where the test harness is tested once that everything is properly setup as it should be ? (it could be even something as involved as the script reference_models/test_config.py which checks that all tiles are there, etc..)
Comment from @kate-harrison  ?

Reply:
Agreed.   Based on the suggestion test_config.py will be invoked in setupClass method   to ensure that setup is configured properly before start the PCR tests.